src/Modules/Guilds/Wrappers/ProposalVoteCardWrapper.tsx

```diff
@@ -0,0 +1,79 @@
+import { useERC20Guild } from 'hooks/Guilds/contracts/useContract';
+import { useProposal } from 'hooks/Guilds/ether-swr/guild/useProposal';
+import useProposalMetadata from 'hooks/Guilds/ether-swr/guild/useProposalMetadata';
+import useSnapshotId from 'hooks/Guilds/ether-swr/guild/useSnapshotId';
+import { useVotingPowerOf } from 'hooks/Guilds/ether-swr/guild/useVotingPowerOf';
+import { useVotingResults } from 'hooks/Guilds/ether-swr/guild/useVotingResults';
+import useVotingPowerPercent from 'hooks/Guilds/guild/useVotingPowerPercent';
+import useTimedRerender from 'hooks/Guilds/time/useTimedRerender';
+import { useWeb3React } from '@web3-react/core';
+import { useTypedParams } from 'Modules/Guilds/Hooks/useTypedParams';
+import { useTransactions } from 'contexts/Guilds';
+import { ProposalVoteCard } from 'Components/ProposalVoteCard';
+
+const ProposalVoteCardWrapper = () => {
+  const { guildId, proposalId } = useTypedParams();
+  const { data: proposal } = useProposal(guildId, proposalId);
+  const { data: proposalMetadata } = useProposalMetadata(guildId, proposalId);
+
+  const voteData = useVotingResults();
+
+  const timestamp = useTimedRerender(10000);
+
+  const { account: userAddress } = useWeb3React();
+  const { data: userVotingPower } = useVotingPowerOf({
+    contractAddress: guildId,
+    userAddress,
+  });
+  const { createTransaction } = useTransactions();

```

Comment:
I think since useTransacions hook is only keeping track of functions via a context we could actually move this into the component itself since it will never change depending on implementation much like how we use toast

Reply:
Hm, it did something else with IPFS, CID, and Orbit-DB as well so I'd need to mock away all of those and engineer it within the test - so I decided to keep it. But I tried at least 🤷

Reply:
Oh my bad, I might have confused this for something else